### PR TITLE
update makefile

### PR DESCRIPTION
**Pull request by MegaByte875**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

### CODE REVIEW

**Review by kqzh — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -106,70 +89,65 @@ run-controller-manager: manifests generate check
...
+docker-build: ## Build docker images.
+	docker build -t "${DOCKER_REPO}/nebula-operator:${IMAGE_TAG}" .
 
 docker-push: ## Push docker images.
 	docker push "${DOCKER_REPO}/nebula-operator:${IMAGE_TAG}"
 
-docker-manifest: ## Build all docker images and push it to registry.
-	bash build.sh ${DOCKER_REPO}/nebula-operator ${IMAGE_TAG}
+ensure-buildx:
+	./hack/init-buildx.sh
```
Comment (wenhaocs) [inline Makefile:99]: Please add +x to this script.

File: Makefile [hunk 2]
```diff
@@ -106,70 +89,65 @@ run-controller-manager: manifests generate check
 run-scheduler: manifests generate check
 	go run -ldflags '$(LDFLAGS)' cmd/scheduler/main.go
 
-docker-build: build ## Build docker images.
-	docker build -t "${DOCKER_REPO}/nebula-operator:${IMAGE_TAG}" images/nebula-operator/
+docker-build: ## Build docker images.
```
Comment (wenhaocs) [inline Makefile:92]: The current `Dockerfile` does not work for `make docker-build`. Please resolve the following error `Step 1/5 : FROM --platform=$BUILDPLATFORM ubuntu:focal
failed to parse platform : "" is an invalid component of "": platform specifier component must match "^[A-Za-z0-9_-]+$": invalid argument`

Comment (MegaByte875) [reply Makefile:92]: https://github.com/vesoft-inc/nebula-operator/pull/236

File: Makefile (hunk 1 above)
Comment (MegaByte875) [reply Makefile:99]: done

**Pull request merged**